### ISSUE TITLE
macOS support

### ISSUE THREAD

**Issue by sylvinus** [marian-m12l/studio#26]
Hello!

Would it be possible to add macOS as a supported platform? I'm happy to report that I got it working (at least the web interface so far).

I just changed the startup script to this:
```
#!/bin/sh
​
DOT_STUDIO=$HOME/.studio
LOCAL_LUNIITHEQUE="$HOME/Library/Application Support/Luniitheque"
​
# Make sure the .studio subdirectories exist
if [ ! -d $DOT_STUDIO/db ]; then mkdir -p $DOT_STUDIO/db; fi
if [ ! -d $DOT_STUDIO/lib ]; then mkdir -p $DOT_STUDIO/lib; fi
if [ ! -d $DOT_STUDIO/library ]; then mkdir -p $DOT_STUDIO/library; fi
​
# Copy Luniistore JARs if needed
if [ ! -e "$DOT_STUDIO/lib/lunii-java-util.jar" ]; then cp "$LOCAL_LUNIITHEQUE/lib/lunii-java-util.jar" $DOT_STUDIO/lib/; fi
if [ ! -e "$DOT_STUDIO/lib/lunii-device-gateway.jar" ]; then cp "$LOCAL_LUNIITHEQUE/lib/lunii-device-gateway.jar" $DOT_STUDIO/lib/; fi
if [ ! -e "$DOT_STUDIO/lib/lunii-device-wrapper.jar" ]; then cp "$LOCAL_LUNIITHEQUE/lib/lunii-device-wrapper.jar" $DOT_STUDIO/lib/; fi
​
java -cp studio-web-ui-0.1.2.jar:lib/*:$DOT_STUDIO/lib/*:. io.vertx.core.Launcher run studio.webui.MainVerticle
```

cheers,

**Comment by marian-m12l** [comment]
Hi,

Thanks for the feedback! I have no way to test macOS support, but I'll gladly add the startup script. I assume the Luniistore (being written with Java) is the same on all platforms anyway.

I'll need to add a third script and fix the doc.

**Comment by marian-m12l** [comment]
Should be fixed on master.

**Comment by marian-m12l** [comment]
Fixed in release 0.1.3
